sys/include/net/sock/dtls.h

```diff
@@ -468,6 +479,31 @@
  * return 0;
  * ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  *
+ * ### Multi-credential handling
+ *
+ * Each sock needs at least one credential tag. Extra credentials can be added
```

Comment:
This might need updating after the changes to allow `CREDMAN_TAG_EMPTY` on `sock_dtls_create`.